src/Compute/Compute.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,115 @@
+Invoke-LiveTestScenario -Name "Creates a virtual machine." -Description "Test create new VM" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+
+    ## VM Account
+    # Credentials for Local Admin account you created in the sysprepped (generalized) vhd image
+    $VMLocalAdminUser = "LocalAdminUser"
+    $VMLocalAdminSecurePassword = ConvertTo-SecureString "Password" -AsPlainText -Force
+    ## Azure Account
+    $LocationName = "westus"
+    # This a Premium_LRS storage account.
+    # It is required in order to run a client VM with efficiency and high performance.
+    $StorageAccount = "Mydisk"
+
+    ## VM
+    $OSDiskName = "MyClient"
+    $ComputerName = "MyClientVM"
+    $OSDiskUri = "https://Mydisk.blob.core.windows.net/disks/MyOSDisk.vhd"
+    $SourceImageUri = "https://Mydisk.blob.core.windows.net/vhds/MyOSImage.vhd"
```

Comment:
Are these URLs correct? I was not able to access these VHDs with the message resource not found.